packages/icons/package.json

```diff
@@ -3,7 +3,6 @@
   "version": "0.2.0",
   "license": "Apache-2.0",
   "main": "src/index.ts",
-  "sideEffects": false,
```

Comment:
Ah I think this is because the Icon component was moved to this package. We can also copy the sideEffects value from core